src/app/_lib/models/test.ts

```diff
@@ -20,13 +20,15 @@ export type ReadingTest = {
 export type ListeningTest = {
   id: string;
   title: string;
+  text: string;
   autioUrl: string;
   questions: Question[];
 }
 
 export type GrammarTest = {
   id: string;
   title: string;
+  text: string;
```

Comment:
Do we need text in Grammar and Listening tests? I thought audio in Listening and just questions in the grammar test would be enough.

Reply:
Maybe we need to add 'instructions' property to every test type. There we can provide instructions for every type of exercise and 'text' could be only relevant for the reading test.